### PR TITLE
#1584: treat textures as sequences

### PR DESCRIPTION
**Pull request by aparna-d2**

### CODE REVIEW

File: hooks/mari/publish_textures.py [hunk 1]
```diff
@@ -93,6 +93,8 @@ def validate(self, task_settings, item):
         :param item: Item to process
         :returns: True if item is valid, False otherwise.
         """
+        # textures should always contain UDIM or tag <UDIM>
+        item.properties["is_sequence"] = True
```
Comment (mbrainerd) [inline hooks/mari/publish_textures.py]: This should be set in the collector, not here

File: hooks/publish.py [hunk 2]
```diff
@@ -354,7 +354,7 @@ def validate(self, task_settings, item):
             )
             return False
 
-        if item.properties.get("is_sequence"):
+        if item.properties.get("is_sequence") and item.properties.get("path"):
```
Comment (mbrainerd) [inline hooks/publish.py]: Feels like this entire block down to line 397 should be wrapped in the `if item.properties.get("path")` conditional.

File: hooks/mari/publish_textures.py (hunk 1 above)
Comment (aparna-d2) [reply hooks/mari/publish_textures.py]: Good point. Changing.

File: hooks/publish.py (hunk 2 above)
Comment (aparna-d2) [reply hooks/publish.py]: I don't think so. `path` is simply the work file path. Even if that doesn't exist (eg. for mari textures), we still need to set the publish path (else block, lines 360-362) and confirm there are no conflicting publishes (rest of the code till 397)

**Pull request merged**